### PR TITLE
Create LOC7376Chachara.xml

### PR DESCRIPTION
**Pull request by GueshSol**

### CODE REVIEW

File: LOC7376Chachara.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <revisionDesc>
+            <change who="GS" when="2023-06-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="dabr">
+                     <placeName xml:id="n1" xml:lang="gez">ጫጨረ፡ ጊዮርጊስ፡</placeName>
+                    <placeName corresp="#n1" xml:lang="gez" type="normalized">Č̣āč̣ra Giyorgis</placeName>
```
Comment (thea-m) [inline LOC7376Chachara.xml:49]: Č̣āč̣ara

File: LOC7376Chachara.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7376Chachara" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Chachara Giyorgis</title>
```
Comment (thea-m) [inline LOC7376Chachara.xml:8]: Č̣āč̣ara

File: LOC7376Chachara.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <change who="GS" when="2023-06-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="dabr">
+                     <placeName xml:id="n1" xml:lang="gez">ጫጨረ፡ ጊዮርጊስ፡</placeName>
+                    <placeName corresp="#n1" xml:lang="gez" type="normalized">Č̣āč̣ra Giyorgis</placeName>
+                </place>
```
Comment (thea-m) [inline LOC7376Chachara.xml:50]: I believe I remember that you can find the exact geographical coordinates of this church on Google maps? Could you include them here?

File: LOC7376Chachara.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LOC7376Chachara.xml:49]: Please be sure that this is the full name of the institution, and not its short "common" variant (the full name frequently includes the element Dabra ..., like Dabra Sahay, Dabra Madhanit or the like).

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]
I think that   ```<placeName xml:id="n2" xml:lang="gez" type="alt">ደብረ፡ ኃይል፡</placeName>
                    <placeName corresp="#n2" xml:lang="gez" type="normalized">Dabra  Ḫāyl</placeName>```
and   ```<relation name="gn:locatedIn" active="LOC7376Chachara"
                        passive="LOC5498Sallam"/>```
can also be inserted into https://github.com/BetaMasaheft/Institutions/pull/73/files

**Comment by thea-m** [comment]
Thank you for your patience, Guesh

**Pull request merged**